src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+li,
+input,
+button,
+textarea {
+  padding: 0;
   margin: 0;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__item {
+  list-style: none;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  /* margin: 0 10px; */
```

Comment:
Please remove the commented code. If it's not used, it might look confusing, especially when it becomes outdated.